pallets/allocations/src/lib.rs

```diff
@@ -18,7 +18,9 @@
 
 #![cfg_attr(not(feature = "std"), no_std)]
 
+#[cfg(feature = "runtime-benchmarks")]
```

Comment:
do you need that flag? It is normally showing up on top of the `benchmarking.rs` file

Reply:
Indeed not required, removed, Done :+1: